src/components/PoolOverview.tsx

```diff
@@ -86,9 +85,9 @@ export default function PoolOverview({
       <div className="left">
         <div className="titleAndTag">
           {isMetapool ? (
-            <ToolTip content={t("metapool")}>
+            <Tooltip title={<React.Fragment>{t("metapool")}</React.Fragment>}>
```

Comment:
This prop won't take just strings 👀 ?

Reply:
We can use `string` and `React component` for this props.
FYI : https://mui.com/components/tooltips/#customization